ai_genomics/pipeline/gtr/gtr_utils.py

```diff
@@ -0,0 +1,22 @@
+# Scripts to explore definitions in the GtR data
+import json
+import logging
+import boto3
+from typing import List, Dict
+
+from toolz import pipe
+
+
+def fetch_gtr(table_name: str) -> List[Dict]:
+    """Fetch a json gtr object"""
+    s3 = boto3.resource("s3")
+    ai_genomics_bucket = s3.Bucket("ai-genomics")
+
+    logging.info(f"Fetching table {table_name}")
+
+    return pipe(
```

Comment:
you could use the load_s3_data function in ai_genomics.getters.data_getters to load gtr

Reply:
but still have this as a wrapper function i.e.

```
from ai_genomics import bucket_name

def fetch_gtr(table_name: str) -> List[Dict]:
 """Fetch a json gtr object"""
return load_s3_data(bucket_name, f"inputs/gtr/{table_name}.json)
```